infra/vsts_agent_ubuntu_20_04_startup.sh

```diff
@@ -95,6 +95,44 @@ useradd \
   vsts
 #add docker group to user
 usermod -aG docker vsts
+# let vsts user mount/unmount cache folders
+echo "/tmp/bazel_cache /home/vsts/.cache/bazel auto rw,user,exec" >> /etc/fstab
+echo "/tmp/disk_cache /home/vsts/.bazel-cache auto rw,user,exec" >> /etc/fstab
```

Comment:
I didn't know you could just mount directories to other places like this, but I will take your word for it that it works.

Reply:
Mounting directories is possible but a bit more complicated; this is mounting files. Specifically, files that have a file system on them.